src/parser/jobs/rpr/modules/Procs.tsx

```diff
@@ -19,13 +19,15 @@ const SEVERITIES = {
 }
 
 const LOST_PROC_POTENCY = 60
+const ENCHANCED_GIBBET_IGNORE_INTERVAL = 23000
```

Comment:
This should be 2300ms (2.3s)
```suggestion
const ENCHANCED_GIBBET_IGNORE_INTERVAL = 2300
```